src/main/fc/fc_msp.c

```diff
@@ -1745,6 +1748,9 @@ static mspResult_e mspFcProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
             gyroConfigMutable()->gyro_soft_notch_hz_2 = sbufReadU16(src);
             gyroConfigMutable()->gyro_soft_notch_cutoff_2 = sbufReadU16(src);
         }
+        if (dataSize > 17) {
```

Comment:
Sorry that the fact that `dataSize` is deprecated is not made clearer, but since it's not a function it would be difficult to find one spot to note this down that then would be seen by all developers. It was mentioned in the past in a bunch of PRs for `fc_msp.c`, but that is obviously not enough, since not all developers seem to read other people's PRs.

'Deprecated' exactly means that something should not be used in new code, albeit it is still used in existing code. If you think about it, it does not make any sense to declare something deprecated, if you then go on and remove all use of it, in this case you should just remove it too.